components/tcf/ui/src/FirstLayer/index.scss

```diff
@@ -85,7 +86,8 @@ $base-class-modal: '#sui-TcfFirstLayerModal';
   }
 
   &-logo {
-    width: $maw-tcf-first-layer-logo;
+    max-width: $maw-tcf-first-layer-logo;
+    height: $h-tcf-first-layer-logo;
```

Comment:
In order to avoid changes on the logo aspect ratio, you should use `object-fit: contain;`.

Reply:
Done! Thanks